CallAutomation_AppointmentReminder/CallAutomation_AppointmentReminder/Program.cs

```diff
@@ -0,0 +1,136 @@
+using Azure.Communication;
+using Azure.Communication.CallAutomation;
+using Azure.Messaging;
+using CallAutomation_AppointmentReminder;
+using Microsoft.Extensions.FileProviders;
+using static CallAutomation_AppointmentReminder.Logger;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddControllers();
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+
+var sourceIdentity = await Identity.CreateUser(builder.Configuration["ConnectionString"]);
+var phoneNumberToAddToCall = builder.Configuration["TargetPhoneNumber"];
+builder.Services.AddSingleton(new CallAutomationClient(builder.Configuration["ConnectionString"]));
+builder.Services.AddSingleton(new CallConfiguration(builder.Configuration["ConnectionString"],
```

Comment:
you can use the `builder.Services.Configure<CallConfiguration>(builder.Configuration.GetSection(nameof(CallConfiguration))` as one line and the binder will populate the object with everything.

...then you can inject `IOptions<CallConfiguration>` as this will also take care of singleton registration and supports things like dynamic updates to the configuration through the provider.